src/sales-channels/asc/ob-store-review.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: 'Review Store Settings'
+title: Store Settings
 ---
 
-After you connect your Amazon Sales Channel store, you should review your store's settings to ensure your product, listing, pricing, and other settings are correct for your store.
+After you create and integrate your Amazon Sales Channel store, you should review your store's default settings to ensure your product, listing, pricing, and other settings are correct for your store.
 
-## Advanced Store Settings
+## Store Settings
 
 1. On the _Admin_ sidebar, go to **Marketing** > _Channels_ > **Amazon**.
 
-1. Select the store you want to review and click <span class="btn">View Store</span>.
+1. Select the store you want to view and click <span class="btn">View Store</span>.
 
-1. On the store dashboard, click a store setting in the left-side menu.
+1. On the store's dashboard, click a store setting in the left-side menu.
```

Comment:
I wouldn't use possessive, as it is bad form for non-person nouns. I've been trying to clean some of those up during editorial passes. The `store dashboard` works just fine.

Reply:
will fix in the next PR.   well require a find and replace, this is used often.